neofetch

```diff
@@ -3374,7 +3365,6 @@ END
     cpu="${cpu//[1-9][0-9]-Core}"
     cpu="${cpu//[0-9]-Core}"
     cpu="${cpu//, * Compute Cores}"
-    cpu="${cpu//Core / }"
```

Comment:
Why was the `Core ` pattern removed here?

Reply:
It turned my Intel Core 2 Duo into an Intel 2 Duo.

Reply:
Oh... okay I just saw that in the pr comment oops 😭